docs/source/mtraining.rst

```diff
@@ -4,6 +4,190 @@
 mTraining Module
 ================
 
-To be written.
+.. contents::
+    :depth: 3
 
-If you'd like to author this topic, please assign yourself `MOTECH-962 <https://applab.atlassian.net/browse/MOTECH-962>`_.
+###########
+Description
+###########
+
+Provides data containers and APIs for defining training courses. The module gives possibility to and track user enrollments
+and course progress. Courses structure can be very diverse thanks to flexibility of the mTraining entities. Structure of the
+training may be created through the API or by using the user interface.
+
+################
+Course Structure
+################
+
+Introduction
+############
+
+Course structure consists of units that will meet the objectives specified by the implementations. Every element of the
+structure besides of ``Questions`` contains following fields.
+
++------------+---------------------------------------------------------------------------------+--------------------+
+|Field       |Description                                                                      |Type                |
++============+=================================================================================+====================+
+|name        |Name of the course unit.                                                         |String              |
++------------+---------------------------------------------------------------------------------+--------------------+
+|state       |Status of the course unit. Assume the following values: ``Active``,              |CourseUnitState     |
```

Comment:
Assume? I don't think it's the word that you wanted to use. It can take one of the values:...